README.md

```diff
@@ -1,3 +1,13 @@
+
+[![Documentation Status](https://readthedocs.org/projects/khiva-python/badge/?version=latest)](https://khiva-python.readthedocs.io/en/latest/?badge=latest)
+[![License: MPL 2.0](https://img.shields.io/badge/License-MPL%202.0-brightgreen.svg)](https://github.com/shapelets/khiva-python/blob/master/LICENSE.txt)  
+[![py364 status](https://img.shields.io/badge/python3.6.4-supported-green.svg)](https://github.com/shapelets/khiva-python/issues)
+Build Master Branch [![Build Status](https://travis-ci.com/shapelets/khiva-python.svg?branch=master)](https://travis-ci.com/shapelets/khiva-python)
+Code Coverance Master Branch: [![codecov.io](https://codecov.io/github/shapelets/khiva-python/coverage.svg?branch=master)](https://codecov.io/github/shapelets/khiva-python)
+Build Develop Branch [![Build Status](https://travis-ci.com/shapelets/khiva-python.svg?branch=develop)](https://travis-ci.com/shapelets/khiva-python)
+Code Coverance Develop Branch [![codecov.io](https://codecov.io/github/shapelets/khiva-python/coverage.svg?branch=develop)](https://codecov.io/github/shapelets/khiva-python)
+
+
 # README #
 This is the KHIVA binding for connecting the Python programming language and the KHIVA library.
```

Comment:
This is the Khiva Python binding, it allows the usage of Khiva library from Python.